components/search_box.js

```diff
@@ -1,8 +1,9 @@
 // from: https://raw.githubusercontent.com/UKHomeOffice/govuk-react/master/components/search-box/src/index.js
-import React from "react";
+import React, { Component } from "react";
 import PropTypes from "prop-types";
 import styled from "react-emotion";
 import SearchIcon from "@material-ui/icons/Search";
+import ClearIcon from "@material-ui/icons/Clear";
```

Comment:
The icon in the designs is different. Were you thinking that would be a later PR?

Reply:
The correct icon is "cancel" in the Navigation section here: https://material.io/tools/icons/?icon=highlight_off&style=round